src/openforms/config/models.py

```diff
@@ -153,7 +153,7 @@ class GlobalConfiguration(SingletonModel):
     )
 
     form_step_previous_text = models.CharField(
-        _("step previous text"),
+        _("previous step"),
```

Comment:
```suggestion
        _("previous step text"),
```

consistency with `form_change_text` and `form_confirm_text`